posthog/models/annotation.py

```diff
@@ -4,8 +4,10 @@
 from django.utils import timezone
 from django_deprecate_fields import deprecate_field
 
+from posthog.models.activity_logging.model_activity import ModelActivityMixin
 
-class Annotation(models.Model):
+
+class Annotation(ModelActivityMixin, models.Model):
```

Comment:
Love a good mixin.

Are there any performance considerations here since this will intercept all saves to the model?

Also should the mixin support deletes?